lib/components/src/navigation/routed_link.test.js

```diff
@@ -6,89 +6,132 @@ const LEFT_BUTTON = 0;
 const MIDDLE_BUTTON = 1;
 const RIGHT_BUTTON = 2;
 
+const createEvent = options => ({
+  button: LEFT_BUTTON,
+  preventDefault: jest.fn(),
+  ...options,
+});
+const render = props => shallow(<RoutedLink {...{ children: 'Content', ...props }} />);
```

Comment:
Well, in that case you don't even use object shorthand notation, so it's even more weird